src/common/snap_types.cc

```diff
@@ -29,6 +33,9 @@ void SnapRealmInfo::dump(ceph::Formatter *f) const
   f->dump_unsigned("seq", seq());
   f->dump_unsigned("parent_since", parent_since());
   f->dump_unsigned("created", created());
+  if (!last_modified.is_zero()) {
+    f->dump_stream("last_modified") << last_modified;
+  }
```

Comment:
Maybe we should dump the `last_modified` even it's zero ?

Reply:
fair enough.